documentation/adr/README.md

```diff
@@ -0,0 +1,54 @@
+# Architecture Decision Records (ADRs)
+
+
+## About ADRs
+
+### Why do we need to document architecture decisions?
+
+Developing software involves making lots of decisions. Documenting these decisions and their outcomes achieves several important goals:
+
+1. **Knowledge Preservation:** Architectural decisions capture the rationale behind important choices made during the design and development process. They serve as a reference point for future developers, allowing them to understand the thinking and context behind the architecture. Without proper documentation, critical knowledge can be lost over time as team members change or move on to other projects.
+2. **Communication and Collaboration:** Documentation helps facilitate communication among team members and stakeholders. It allows the product owner, developers, and other stakeholders to share a common understanding of the software's architecture, its components, and the reasoning behind specific decisions.
+3. **Decision Transparency:** ADRs provide greater insight into the choices made throughout the project. It helps team members and stakeholders understand the factors considered, alternatives evaluated, and the trade-offs involved in the decision-making process. This transparency fosters trust, enables better collaboration, and allows for informed discussions and feedback.
+4. **Risk Management:** Documenting key decisions also allows project stakeholders to assess the potential impact of architectural changes, identify dependencies, and evaluate the consequences of modifying specific components. By having a clear understanding of the trade-offs associated with each option, teams can make more informed decisions and mitigate potential risks associated with changes or enhancements.
+5. **Improved Maintainability:** Software projects evolve over time due to changing requirements, new features, or technological advancements. Documentation ensures that future developers can understand the architecture's design principles, patterns, and constraints. It enables future maintainers to modify or extend the codebase in a consistent and coherent manner, reducing the risk of introducing unintended side effects or architectural drift.
```

Comment:
Maybe "Maintainability and Adaptability."? To indicate that it also helps us be more confident in changing our minds later. 

I remember on CEJST we switched from a mapping framework (whose name I'm currently blanking on) back to MapLibre after playing with the former for a while. By revisiting our decision log, it was very easy to see that given our new information, MapLibre was the best choice.

Reply:
Yeah I remember that switch! It was really helpful to see the history of that decision. 

I changed this to `Improved Maintainability and Adaptability` in [this commit](https://github.com/HHS/grants-api/pull/15/commits/0e4207784335a5e4e8a6da53ce7fa979b9ea075e)